.vscode/launch.json

```
@@ -43,7 +43,7 @@
       "sourceMaps": true,
       "cwd": "${workspaceFolder}/src/examples/mailchimp"
     },
-    {
+     {
```

Extremely pedantic, I know:
```suggestion
    {
```

---

Hmm thats annoying let me tweak my vscode settings.